src/lib/__tests__/RoutingManager-test.ts

```diff
@@ -402,8 +438,8 @@ describe('RoutingManager', () => {
         },
       });
 
-      const fakeSearchBox = createFakeSearchBox();
-      const fakeHitsPerPage = createFakeHitsPerPage();
+      const fakeSearchBox: any = createFakeSearchBox({});
```

Comment:
`any` is used because we dynamically create a `refine()` function on the widget to test it. This is not part of the original widget definition.